monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -1,41 +1,43 @@
 groups:
   - name: cluster health
     rules:
-      - alert: health error
+      - alert: CephHealthError
         expr: ceph_health_status == 2
         for: 5m
         labels:
           severity: critical
           type: ceph_default
-          oid: 1.3.6.1.4.1.50495.15.1.2.2.1
+          oid: 1.3.6.1.4.1.50495.1.2.1.2.1
         annotations:
+          summary: Cluster is in an ERROR state
           description: >
             Ceph in HEALTH_ERROR state for more than 5 minutes.
             Please check "ceph health detail" for more information.
 
-      - alert: health warn
+      - alert: CephHealthWarning
         expr: ceph_health_status == 1
         for: 15m
         labels:
           severity: warning
           type: ceph_default
-          oid: 1.3.6.1.4.1.50495.15.1.2.2.2
         annotations:
+          summary: Cluster is in a WARNING state
           description: >
             Ceph has been in HEALTH_WARN for more than 15 minutes.
             Please check "ceph health detail" for more information.
 
   - name: mon
     rules:
-      - alert: Monitor down, quorum is at risk
+      - alert: CephMonDownQuorumAtRisk
```

Comment:
I realized that this might have UX impact since I think this is the title of the notification we display for any alert event... So, speaking personally, `Monitor down, quorum is at risk` would be far more readabable than `CephMonDownQuorumAtRisk`... Perhaps we can display the new 'summary' annotation instead?

Reply:
Yep - I think we should provide both. The alert name for quick reference (find) in the file, and the summary as the one liner description that describes the event.